drf_roles/mixins.py

```diff
@@ -19,7 +18,14 @@ class RoleError(Exception):
 class RoleViewSetMixin(object):
     """A ViewSet mixin that parameterizes DRF methods over roles"""
     _viewset_method_registry = set(getattr(settings, "VIEWSET_METHOD_REGISTRY", DEFAULT_REGISTRY))
-    _role_groups = set(getattr(settings, "ROLE_GROUPS", DEFAULT_GROUPS))
+
+    def __init__(self, **_kwargs):
+        groups = []
+        if hasattr(settings, "ROLE_GROUPS"):
```

Comment:
Don't need this guard if you are providing a default to `getattr`.

Reply:
7fe82502